packages/block-library/src/index.js

```diff
@@ -309,6 +312,22 @@ export const registerCoreBlocks = (
 ) => {
 	blocks.forEach( ( { init } ) => init() );
 
+	// Auto-register PHP-only blocks with ServerSideRender
+	if ( window.__unstableAutoSSRBlocks ) {
+		window.__unstableAutoSSRBlocks.forEach( ( blockName ) => {
+			registerBlockType( blockName, {
+				title: blockName,
+				category: 'text',
+				edit: ( { attributes } ) =>
```

Comment:
While discussing this with @jorgefilipecosta he proposed a nice idea that we could try to explore separately:

- Add a "default edit" setting to `BlockEditorSettings` that could be built into the `editor` package.
- Have `unstable__bootstrapServerSideBlockDefinitions` auto register the blocks that have the "auto-register" flag set to true.

We might remove the need for the global variable with this.